ElementX/Sources/Screens/RoomScreen/RoomScreenCoordinator.swift

```diff
@@ -115,4 +117,27 @@ final class RoomScreenCoordinator: CoordinatorProtocol {
 
         navigationStackCoordinator.push(coordinator)
     }
+
+    private func displayReportContent(for itemId: String) {
+        let navCoordinator = NavigationStackCoordinator()
```

Comment:
Lets not abbreviate please :)
```suggestion
        let navigationCoordinator = NavigationStackCoordinator()
```